listenbrainz/webserver/static/js/src/user/Listens.tsx

```diff
@@ -774,111 +763,109 @@ export default class RecentListens extends React.Component<
                 {listens.length < this.expectedListensPerPage && (
```

Comment:
Same as recent listens. looks redundant.

Reply:
In this case we can have some listens but not enough to fill the page (i.e new user only has 5 listens to show)
This is probably why RecentListens still had something similar still in there, I must have modified the condition to remove expectedListensPerPage, but it didn't make sense in Recent page.